### ISSUE TITLE
Error adding http.alias

### ISSUE THREAD

**Issue by aaroncarlucci** [aws/copilot-cli#2510]
I used the new feature in 1.8 to add a custom URL using the http.alias configuration key. My app was originally created using a --domain flag and I wanted to make an alternate, production URL that the services could be accessed at.

I added the following to the service manifest.yml:

```
environments:
  prod:
    http:
      alias: 'dashboard.mydomain.com'

```
When I ran the the copilot deploy for my service, the CloudFormation stack for the service failed to update because it tried to update the stack for the prod environment, which failed with the message:

`Resource handler returned message: "Certificate ARN '2021/06/24/[$LATEST]a9e0c41da27c47588081ca8ca0b22e67' is not valid (Service: ElasticLoadBalancingV2, Status Code: 400, Request ID: ded19519-87b7-4518-beb2-3dd23442fa71, Extended Request ID: null)" (RequestToken: a273d8bd-a9d7-6c9a-4d60-0450932532f9, HandlerErrorCode: InvalidRequest)`

I'm not sure why what looks like a CloudWatch log group or instance is being used as an ACM ARN.

Something to note - I don't have an existing hosted zone for this domain. Does this need to be created, and if so, perhaps adding more to the documentation on the exact requirements for this configuration option are in order. Either way, the error looks unrelated and very ambiguous.

I also tried by putting the http.alias configuration in the main part of the manifest config (not overridden by environment) but got the same result.

**Comment by iamhopaul123** [comment]
Hello @acarlton. Sorry for the confusion. It is really strange that the certificate ARN is `2021/06/24/[$LATEST]a9e0c41da27c47588081ca8ca0b22e67`. Just to double check does your app account own `mydomain.com` this root domain?

> perhaps adding more to the documentation on the exact requirements for this configuration option are in order.

Yes we are adding more to the doc and a validation in the manifest so as to fail early in https://github.com/aws/copilot-cli/pull/2482, which should be included in next release.

**Comment by aaroncarlucci** [comment]
@iamhopaul123 thanks for the reply. I am confused what you mean by my app account "owning" the domain. Does that mean the DNS is registered in Route53, or does it mean a hosted zone is created for that domain?

The answer to both of these questions is "no", so I'm assuming I need something. Would you please specify?

**Comment by iamhopaul123** [comment]
Hello @acarlton. Like you said "My app was originally created using a --domain flag", so by "owning" the domain I meant the alias you specify should be using the same domain you specified with `--domain` flag when you created the app (please refer [here](https://github.com/aws/copilot-cli/blob/49c6b982b95bd769ae732ac58c557cb42eec02b4/site/content/docs/developing/domain.en.md)).

Sorry right now we don't allow certificate import but ideally in future you can validate any certificate on your own and import them when you create the env. That way you can use whatever alias you want.

**Comment by aaroncarlucci** [comment]
@iamhopaul123 Ahhh -- indeed, no, the alias I am trying to use does not belong to the domain I setup the application with. I would need the future functionality you describe of importing validated ACM certificates. Looking forward to that.

Thanks for the clarification -- I hadn't seen the documentation page you referenced, I was looking at the one-liner in the manifest.yml documentation for Load Balanced Web Services.

**Comment by iamhopaul123** [comment]
@acarlton sure glad to help! I think https://github.com/aws/copilot-cli/pull/2482 should be addressing this issue as well!

> I would need the future functionality you describe of importing validated ACM certificates. Looking forward to that.

❤️

**Comment by iamhopaul123** [comment]
Heya we fixed this in our latest release [v1.8.2](https://github.com/aws/copilot-cli/releases/tag/v1.8.2). Thank you for reporting this issue 🎉
